[FollowUp] Remove !important rule

moiikana opened the pull request:
-  using `!important` in CSS is often a sign of a code smell when writing CSS and we try to avoid it,
    because it means there might was trouble with the CSS's specificity rules (
    see e.g. https://csswizardry.com/2012/11/code-smells-in-css/ -> "Reactive !important")
	`!important` should only be used in very exceptional cases
    
- exchange the p tag (which have strong rather global css rules) with a normal div tag
    (that is easier to style, we don't have to overwrite p rules)

gbirke commented:
Thank you so much for tackling this! I don't know if this is already meant to be reviewed, but I had a quick look and noticed the following things:

* desktop VAR is not changed yet, but that's probably because of the new design
* In the desktop wp.de banner, the text is a bit too close to the banner border  
<img width="498" alt="Bildschirmfoto 2020-12-08 um 18 15 39" src="https://user-images.githubusercontent.com/223326/101518336-12ddf580-3982-11eb-925b-6661ada470d6.png">

* In the mobile (ipad both languages, mobile EN) the border radius of the bar does not fit with the border radius of its frame. I think this is not intentional, it's just that at some point we changed the outer one radius to make it look nicer and forgot how the two radii will look when they are adjacent. But I think @KaiNissen has to chime in here to tell us what the final design should be.  
<img width="274" alt="Bildschirmfoto 2020-12-08 um 18 16 37" src="https://user-images.githubusercontent.com/223326/101518695-8d0e7a00-3982-11eb-9eb3-7e5a53fcb7f4.png">

moiikana commented:
thanks @gbirke, I hadn't been working on the WPDE banners at the time of your comment, that is fixed now.
for the filling shape: according to Kai this is ok I think

moiikana committed:
Remove !important CSS rule

- !important in CSS is usually a sign of a code smell when writing CSS,
because it means there might was trouble with the CSS's specificity rules (
see e.g. https://csswizardry.com/2012/11/code-smells-in-css/ -> "Reactive !important")

- exchange the p tag (which have strong rather global css rules) with a normal div tag
(that is easier to style, we don't have to overwrite p rules)

Abban commented:
Getting rid of the `!important` attribute here is the correct thing to do, but replacing the paragraphs with divs isn't, as it's not semantic. Instead the paragraphs should kept and the margin overidden using higher css specificity.
